zbus/Cargo.toml

```diff
@@ -140,6 +140,7 @@ tracing-subscriber = { version = "0.3.18", features = [
 ], default-features = false }
 tempfile = "3.10.1"
 criterion = "0.5.1"
+blocking = "1.6.1"
```

Comment:
Is this new? I don't recall see thing change before. 

> vsock crate is not async, but the following added p2p test needs to have
both client&server side async connect to each other.

We've abstractions for both running async in sync (block_on) and sync in async (spawn_blocking). Please make use of those instead. The spawn_blocking actually uses `blocking` when `async-io` (default) is enabled. `spawn_blocking` is currently `pub(crate)` though but feel free to change that to `pub`.

Reply:
done